lib/linkscape/client.rb

```diff
@@ -179,8 +179,19 @@ def anchorMetrics(*args)
       Linkscape::Request.run(@options.merge(options))
     end
 
+    def metadata(*args)
+      options = Hash === args.last ? args.pop.symbolize_keys : {}
+      command = args.first ? args.shift : options[:command]
```

Comment:
`command` seems like a bit of an odd name for this parameter. Also, I would probably use a required argument for the first parameter, but it appears all of the other methods use a single splat, so I don't feel too strongly about it.

Honestly, this whole gem could use a bit of refactoring polish, but no one has spent the time on it.